examples/jsm/controls/TrackballControls.js

```diff
@@ -12,7 +12,7 @@ const _endEvent = { type: 'end' };
 
 class TrackballControls extends EventDispatcher {
 
-	constructor( object, domElement ) {
+	constructor( object, domElement, target ) {
```

Comment:
IMO, modifying the ctor signature for is inappropriate. You can assigning a custom target like so and achieve the same effect:
```js
controls.targert = myTarget;
```

Reply:
```js
controls.target.set( x, y, z );
```
or
```js
controls.target = myObject.position;
```